playbooks/roles/wordpress/tasks/nginx.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
@Agrendalath, don't you think a lot of this is covered by the existing `nginx-proxy` role? Is there a reason why this should be added separately?

Reply:
We are running them on the same port, so I'm not sure what we would gain by setting up an `nginx-proxy` here. Am I missing something?

Reply:
@Agrendalath, nginx acts as a proxy in this case because the actual Wordpress code is run on the php-fpm servers. So my question was about using the `nginx-proxy` role to do all the below tasks instead of repeating it all here.

Reply:
Yep, we are using it as a proxy, but we also host files with it, so it's not only the proxy. We could reuse this role, but we would need to overwrite its template to be able to use it for our needs (which would greatly decrease its readability) or overwrite the role itself to allow custom templates (and risk breaking some other roles that depend on this one).

Reply:
@Agrendalath, we should leave it as-is and leave this improvement and the investigation for it, for a later task